src/ui/Footer.vue

```diff
@@ -0,0 +1,212 @@
+          <!-- <a-select
+            default-value="LaTeX"
+            style="width: 100px"
+            size="small"
+            @change="docActions.handleChangeOUTformat"
+            :getPopupContainer="trigger => trigger.parentNode"
+          >
+            <a-select-option value="string">
+              String
+            </a-select-option>
+            <a-select-option value="LaTeX">
+              LaTeX
+            </a-select-option>
+          </a-select> -->
+
+          <div :style="{ width: '20px' }" />
+          <!-- Virtual Keyboard -->
+          <!-- <a-tooltip>
+            <template #title> Popout Keyboard </template>
+            <a-button size="small" @click="appActions.togglevirtualkb">
+              <AppstoreOutlined />
+            </a-button>
+          </a-tooltip> -->
+
+          <div :style="{ width: '20px' }" />
+
+          <!-- ScratchPad -->
+          <!-- <a-tooltip>
+            <template #title> Extended Work Area </template>
+            <a-space>
+              <ExportOutlined />
+              <a-switch size="small" @change="appActions.handleChangeScratchPad" />
+            </a-space>
+          </a-tooltip> -->
+
+          <!-- <div :style="{ width: '20px' }" /> -->
+
+          <!-- Zoom -->
+          <!-- <a-icon type="zoom-out" class="vert-icon" />
+          <a-slider
+            id="test"
+            :min="50"
+            :max="150"
+            :step="25"
+            :default-value="100"
+            :style="{ width: '100px', marginTop: '6px', marginBottom: '6px' }"
+            disabled
+          />
+          <a-icon type="zoom-in" class="vert-icon" /> -->
+
+          <div :style="{ width: '20px' }" />
+        </a-space>
+      </a-col>
+    </a-row>
+  </footer>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, reactive, inject, watch } from 'vue'
+import { ExportOutlined, CalculatorOutlined, AppstoreOutlined } from '@ant-design/icons-vue'
+import * as UI from './ui'
+import * as Notification from './notification'
+import { Icon } from './icon'
+import { cas } from './../model/cas'
+
+function useDocActions() {
+  function handleChangeNformat() {
+    return
+  }
+  function handleChangeOUTformat(value: any) {
+    return
+  }
+  function handleChangeDecimalPlaces(value: any) {
+    return
+  }
+  function updateDocOptions() {
+    return
+  }
+  return {
+    handleChangeNformat,
+    handleChangeOUTformat,
+    handleChangeDecimalPlaces,
+  }
+}
+function useAppActions() {
+  function handleChangeScratchPad(value: any) {
+    return
+  }
+  function togglevirtualkb() {
+    return
+  }
+  function compute() {
+    return
+  }
+  return {
+    handleChangeScratchPad,
+    togglevirtualkb,
+    compute,
+  }
+}
+
+export default defineComponent({
+  components: {
+    ExportOutlined,
+    CalculatorOutlined,
+    AppstoreOutlined,
+    Icon,
+  },
+  props: {},
+  setup(props, context) {
+    // TODO: This whole file is Garbage, clean this up.
```

Comment:
Sounds good, should we leave that to a future pull request? While this file is a mess, it's at least nicely contained mess

Reply:
Yea, the footer needs a lot of work. I'll save it for future PR's since most of its features will tie in with the CAS.